views/partials/dashboards/remove-form-confirm.hbs

```diff
@@ -58,11 +66,24 @@
   </div>
   <div class="remove-dashboard-form-error --general"></div>
   <div class="flx">
-    <a
-      href="#"
-      class="js-remove-confirm remove-dashboard-confirm-submit jst-cntr flx btn-blue-primary"
-      role="button"
-    >{{getConfirmSubmitText}}</a>
+    {{#if this.acctInfo}}
+      {{#if this.allowEditRemovalInfo}}
+      <a
+        href="#"
+        class="js-remove-confirm remove-dashboard-confirm-submit jst-cntr flx btn-blue-primary"
+        role="button"
+      >{{getConfirmSubmitText}}</a>
+      
+      {{else}}
+      <p class="remove-dashboard-conirm-edit">{{getRemoveString "remove-form-change-contact-msg"}}: <a href="mailto:{{ getRemoveString "remove-about-support-email"}}">{{ getRemoveString "remove-about-support-email"}}</a></p>
```

Comment:
typo? Should `remove-dashboard-conirm-edit` be `-confirm-`?

Reply:
Class should be removed. Thanks!